example-get-started/generate.sh

```diff
@@ -109,7 +110,8 @@ dvc run -n evaluate \
         -d src/evaluate.py -d model.pkl -d data/features \
         -M scores.json \
         --plots-no-cache prc.json \
-        python src/evaluate.py model.pkl data/features scores.json prc.json
+        --plots-no-cache roc.json \
+        python src/evaluate.py model.pkl data/features scores.json prc.json roc.json
 dvc plots modify prc.json -x recall -y precision
```

Comment:
do we need to do `dvc plots modify` for the second plot?

Reply:
Yeah, and I need to update the docs here, too. Unfortunately, it adds a little complication in that it's no longer possible to plot everything with one line.